src/interpreter/InterpretAdditionalProperties.ts

```diff
@@ -23,9 +23,11 @@ export default function interpretAdditionalProperties(
   if (typeof schema === 'boolean' || isModelObject(model) === false) {
     return;
   }
+  const ignoreAdditionalProperties =
+    interpreterOptions.ignoreAdditionalProperties || false;
   const additionalPropertiesModel = interpreter.interpret(
     schema.additionalProperties === undefined
-      ? true
+      ? !ignoreAdditionalProperties
```

Comment:
In order to not overcomplicate stuff, how about we just ignore the interpretation instead of interpreting it with `false` 🤔 ? 
```suggestion
  const ignoreAdditionalProperties = interpreterOptions.ignoreAdditionalProperties || false;
  if (typeof schema === 'boolean' || isModelObject(model) === false || ignoreAdditionalProperties) {
    return;
  }
  const additionalPropertiesModel = interpreter.interpret(
    schema.additionalProperties === undefined
      ? true
```

Reply:
The logic has been simplified.